frontend/app/styles/app.css

```diff
@@ -20,8 +40,10 @@ li a {
 
 /* Validation-errors */
 .validation-errors {
-  color: red;
-  font-size: small;
+  width: 100%;
+  margin-top: 0.25rem;
+  font-size: 90%;
```

Comment:
In my opinion, validation errors look good enough like this. I remember @psunix saying, we might want them to stick out more?